app/bundles/CoreBundle/EventListener/BuildJsSubscriber.php

```diff
@@ -137,6 +137,30 @@ public function onBuildJs(BuildJsEvent $event)
     return response;
 };
 
+MauticJS.isPixelLoaded = function() {
+    if (typeof MauticJS.trackingPixel !== 'undefined') {
+        return /co/.test(MauticJS.trackingPixel);
```

Comment:
the `/co/.test` is testing the value of the tracking pixel to see if it contains the letters 'co', so this check will fail. When we run the /co/.test on the document.readState, it's checking the ready state value for the first distinct characters of 'complete', as compared to the other values it could have (loading, interactive, complete).